src/main/java/org/jenkinsci/bytecode/helper/TypeHierarchyReader.java

```diff
@@ -0,0 +1,432 @@
+/***
```

Comment:
The following changes have been made to this class from the upstream source.
* class is no longer public
* changed imports to use `org.kohsuke.asm5` packages for asm
* added a comment to point to the upstream source